<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KCGL</class>
 <widget class="QWidget" name="KCGL">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1375</width>
    <height>779</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,3,2,20">
   <property name="topMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>库存管理</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="minButton" name="toolButton">
       <property name="minimumSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="minimumSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{	
	background-image: url(:/images/max-min_first.png);
	background-repeat:no-repeat;
	background-position:center center;
	border:none;
}
QPushButton:hover
{
	background-repeat:no-repeat;
	background-position:center center;
	background-image: url(:/images/max-min_after.png);
	border:none;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="closeButton" name="toolButton_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>70</height>
      </size>
     </property>
     <property name="title">
      <string>工具栏</string>
     </property>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>55</width>
        <height>54</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/images/增加员工.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_wlbh">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>55</width>
        <height>54</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/images/物料编号.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_daoru">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>51</width>
        <height>54</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/images/库存导入.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_daochu">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>10</y>
        <width>51</width>
        <height>54</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/images/库存导出.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>55</height>
      </size>
     </property>
     <property name="title">
      <string>搜索栏</string>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>20</y>
        <width>54</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>物料名称：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_name">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>20</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>20</y>
        <width>54</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>规格型号：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_ID">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>20</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>20</y>
        <width>54</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>供应商：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_privoder">
      <property name="geometry">
       <rect>
        <x>1062</x>
        <y>20</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>1180</x>
        <y>20</y>
        <width>54</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>项目号：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_proID">
      <property name="geometry">
       <rect>
        <x>1230</x>
        <y>20</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>20</y>
        <width>54</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>物料编号：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Num">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_serialID">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>20</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>检验报告编号：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_serialID">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>20</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>20</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(235, 235, 235);</string>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
      <property name="currentText">
       <string>所有仓位</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>20</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>仓位：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>检验日期：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_inTime">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_time">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>20</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(235, 235, 235);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="arrowType">
       <enum>Qt::DownArrow</enum>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="toolTip">
      <string>右键可查看修改删除库存信息</string>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(216, 235, 233);</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>closeButton</class>
   <extends>QToolButton</extends>
   <header>closebutton.h</header>
  </customwidget>
  <customwidget>
   <class>minButton</class>
   <extends>QToolButton</extends>
   <header>minbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
